<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayOptions</class>
 <widget class="QWidget" name="DisplayOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Display Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="imageFirstCheckBox">
       <property name="text">
        <string>1st image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="imageFirstLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="imageFirstButton">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="imageSecondCheckBox">
       <property name="text">
        <string>2nd image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="imageSecondLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="imageSecondButton">
       <property name="text">
        <string>Select</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="blinkingCheckBox">
     <property name="text">
      <string>Blinking</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="timeOnLabel">
       <property name="text">
        <string>Time On:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="timeOnSpinBox">
       <property name="suffix">
        <string> msec</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>2550</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="timeOffLabel">
       <property name="text">
        <string>Time Off:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="timeOffSpinBox">
       <property name="suffix">
        <string> msec</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>2550</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="brightLabel">
       <property name="text">
        <string>Bright:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="brightSlider">
       <property name="maximum">
        <number>15</number>
       </property>
       <property name="value">
        <number>15</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>imageFirstCheckBox</tabstop>
  <tabstop>imageSecondCheckBox</tabstop>
  <tabstop>blinkingCheckBox</tabstop>
  <tabstop>timeOnSpinBox</tabstop>
  <tabstop>timeOffSpinBox</tabstop>
  <tabstop>brightSlider</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
